src/services/EvalFactor.jl

```diff
@@ -424,7 +424,7 @@ end
 function evalPotentialSpecific( Xi::AbstractVector{<:DFGVariable},
                                 ccwl::CommonConvWrapper{Mixture{N_,F,S,T}},
                                 solvefor::Symbol,
-                                measurement::Tuple=(Vector{Vector{Float64}}(),);
+                                measurement::Vector{<:Tuple}=Tuple[];#FIXME
```

Comment:
```suggestion
                                measurement::AbstractVector{<:Tuple}=Tuple[];#FIXME
```

Reply:
Hi, I don't have any local changes, was going to spend some time on it now, so let me know when you are done and we can merge alll the review comments before I look